packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -275,6 +275,21 @@ const createDefaultImplementation = ({ strapi, db, eventHub, entityValidator })
 
     return db.query(uid).load(entity, field, loadParams);
   },
+
+  loadPages(uid, entity, field, params = {}) {
+    if (!_.isString(field)) {
+      throw new Error(`Invalid load. Expected ${field} to be a string`);
+    }
+
+    const { attributes } = strapi.getModel(uid);
+    const attribute = attributes[field];
+
+    if (!attribute || attribute.type !== 'relation') {
```

Comment:
Should we check the type of relation here too? (as per entity-repository.js line 123)

Reply:
done :) I refactored a bit the relational utils at the same time